tests/test_adapter.py

```diff
@@ -21,3 +22,24 @@ def test_config_identical(model_size, lit_llama):
         llama_model = llama.LLaMA.from_name(model_size)
         adapter_model = llama_adapter.LLaMA.from_name(model_size)
         assert llama_model.lm_head.weight.shape == adapter_model.lm_head.weight.shape
+
+
+def test_adapter_load_gating_factor():
```

Comment:
Windows is failing. This might fix it

```suggestion
def test_adapter_load_gating_factor(lit_llama):
```